tst/testinstall/pperm.tst

```diff
@@ -4117,6 +4117,11 @@ true
 gap> OnTuples([1], PartialPerm([])) = [];
 true
 
+#
+gap> SetUserPreference("PartialPermDisplayLimit", 1);
+gap> PartialPerm([1], [2]);
+<partial perm on 1 pts with degree 1, codegree 2>
```

Comment:
Note: if (when) this PR is merged, #4050 will need to be rebased and updated to change this line to
```suggestion
<partial perm on 1 pt with degree 1, codegree 2>
```